src/status_im/ui/screens/browser/views.cljs

```diff
@@ -103,12 +105,65 @@
        :accessibility-label :modal-chat-button}
       [icons/icon :main-icons/message]]]))
 
+(def resources-to-permissions-map {"android.webkit.resource.VIDEO_CAPTURE" :camera
+                                   "android.webkit.resource.AUDIO_CAPTURE" :record-audio})
+
+(views/defview request-resources-panel [resources url]
+  [react/view styles/blocked-access-container
+   [react/view styles/blocked-access-icon-container
+    [icons/icon :main-icons/camera styles/blocked-access-camera-icon]]
+   [react/view styles/blocked-access-text-container
+    [react/text {:style styles/blocked-access-text}
+     (str url " " (i18n/label :t/page-would-like-to-use-camera))]]
+   [react/view styles/blocked-access-buttons-container
+    [react/view styles/blocked-access-button-wrapper
+     [quo/button
+      {:theme    :positive
+       :style    styles/blocked-access-button
+       :on-press (fn []
+                   (components.permissions/request-permissions
+                    {:permissions (map #(get resources-to-permissions-map %) resources)
```

Comment:
`resources-to-permissions-map` is never `nil` so it safe to go with  `(map resources-to-permissions-map resources)`